google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/AppEngineServerStartupPolicy.java

```diff
@@ -38,17 +37,14 @@
 public class AppEngineServerStartupPolicy implements JavaCommandLineStartupPolicy {
 
   public JavaParameters createCommandLine(CommonModel commonModel) throws ExecutionException {
-    final AppEngineServerData data = (AppEngineServerData) commonModel.getApplicationServer()
-        .getPersistentData();
-    final AppEngineSdk sdk = data.getSdk();
-    if (StringUtil.isEmpty(sdk.getSdkHomePath())) {
+    final CloudSdkService sdkService = CloudSdkService.getInstance();
+    if (StringUtil.isEmpty(sdkService.getCloudSdkHomePath())) {
       throw new ExecutionException("Path to App Engine SDK isn't specified");
```

Comment:
this class gets removed by https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/811 so I didn't bother